fastlane/Fastfile

```
@@ -0,0 +1,59 @@
+  desc "Build and Distribute Debug APK to Firebase"
+  lane :build_and_distribute_debug do |options|
+    # Build the debug variant
+    gradle(
+      task: "assemble#{options[:variant]}Debug"
+    )
+
+    # Distribute to Firebase App Distribution for Stag and UAT
+    firebase_app_distribution(
+      app: ENV['FIREBASE_APP_ID'],
+      firebase_cli_token: ENV['FIREBASE_CLI_TOKEN'],
+      groups: "#{ENV['TESTER_GROUPS']}",
```

rubocop (1.69.1)

[convention] Prefer `to_s` over string interpolation.

(Style/RedundantInterpolation)